JavaScript/JavaScriptSDK/Logging.ts

```diff
@@ -52,28 +67,51 @@
         /**
          * This method will throw exceptions in debug mode or attempt to log the error as a console warning.
          * @param severity {LoggingSeverity} - The severity of the log message
-         * @param message {string} - The log message.
+         * @param message {_InternalLogMessage} - The log message.
          */
-        public static throwInternalNonUserActionable(severity: LoggingSeverity, message: string) {
+        public static throwInternalNonUserActionable(severity: LoggingSeverity, message: _InternalLogMessage) {
             if (this.enableDebugExceptions()) {
                 throw message;
             } else {
-                this.warnToConsole(message);
-                this.logInternalMessage(severity, this.AiNonUserActionablePrefix + message);
+                if (typeof (message) !== "undefined" && !!message) {
+                    if (typeof (message.message) != "undefined") {
+                        if (typeof (message.properties) !== "undefined" && !!message.properties) {
```

Comment:
Looks inconsistent with similar check in pollInternalLogs in Initialization.ts, am I missing something?